src/app/shared/components/template/components/audio/audio.component.html

```diff
@@ -1,65 +1,73 @@
-<div class="container-player margin-t-large">
-  <div class="top-row">
-    <div class="title_and_help">
-      <h3 *ngIf="params.title">{{ params.title }}</h3>
-      <ion-icon
-        *ngIf="params.helpText"
-        [pTooltip]="params.helpText"
-        tooltipPosition="left"
-        tooltipEvent="click"
-        name="help-circle-outline"
-        [src]="params.helpIconAsset"
-        class="audio-help"
-      ></ion-icon>
+<div class="container-player margin-t-large" [attr.data-variant]="params.variant">
+  @if (params.title) {
+    <div class="top-row">
+      <h3>{{ params.title }}</h3>
+      @if (params.variant.includes("large") && params.showInfoButton) {
+        <div class="info-icon-container">
+          <ion-icon
+            name="document-text-outline"
+            [src]="params.infoIconAsset | plhAsset"
+            (click)="clickInfo()"
+          ></ion-icon>
```

Comment:
It's a quirk (feature?) of the `ion-icon` element that the "name" attribute will be used unless a value is provided to the "src" attribute, in which case the latter will be used. It would probably be clearer to use separate `if` statements, but I've used this pattern repeatedly in the template so I think it might be simpler to just add an explanatory comment

Reply:
The other benefit of using an `if` statement is then not passing the undefined value to the `plhAsset` pipe, so yeah I think probably better to just use separate `if` statements for each case and not rely on the unclear handling.